### PR TITLE
WIP: submission turbogit

### PR DESCRIPTION
**Pull request by b4nst**
#### Description
Add new port [turbogit](https://github.com/b4nst/turbogit)

###### Type(s)

- [X] enhancement

###### Tested on

macOS 10.14.6 18G4032

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?- [ ] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by cjones051073** [comment]
@b4nst many thanks for submission. Just for future reference though its bad practise to use your master branch for a PR, as if you commit anything else to it unrelated it will appear here. In future please fork a feature branch to use for PRs to avoid this.

**Comment by b4nst** [comment]
Thanks for your feedback. That makes sense, I will keep that in mind for next time.

File: devel/turbogit/Portfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        b4nst turbogit 1.0.0 v
+name                turbogit
```
Comment (cjones051073) [inline devel/turbogit/Portfile:7]: This line should be removed, the GitHub portgroup sets the name correctly.

File: devel/turbogit/Portfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+categories          devel
+platforms           darwin
+supported_archs     x86_64
+license             MIT
+maintainers         {b4nst @b4nst} openmaintainer
+
+description         Your git boat friend
+long_description    Cli tool built to help you deal with your day-to-day git work.
+
+homepage            https://github.com/b4nst/turbogit
```
Comment (cjones051073) [inline devel/turbogit/Portfile:17]: The GitHub portgroup sets this correctly so this line should be removed.

**Comment by b4nst** [comment]
I need to rework the Makefile and add new asset for it to be build from source. I can do it for next release and update the Portfile then.

**Comment by cjones051073** [comment]
Can you give any osrt of timeline on when the changes needed to build from source might be available ?

**Comment by b4nst** [comment]
@cjones051073 Probably one or 2 weeks

**Comment by cjones051073** [comment]
Ok. Is there a pressing need for this now, or could it wait until it could be updated to build from source ?

**Comment by b4nst** [comment]
Nope, you're right it can wait the next release to be built from source. See you next week! Thanks for all your help/time.

**Comment by cjones051073** [comment]
Thanks, but there was no need to close this. We should keep it open as WIP, to preserve the discussion, and then when you are ready just force push the new update to the branch here.

**Comment by b4nst** [comment]
Ok @cjones051073 here is the version built from source.

**Pull request merged**